### PR TITLE
Live 63 live 65 adding user to db

### PR DESCRIPTION
**Pull request by pkrucz00**
Adding CRUD operations for user and persisting information about him on login

### CODE REVIEW

File: test/membrane_live_web/controllers/user_controller_test.exs [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+  describe "delete user" do
+    setup [:create_user]
+
+    test "deletes chosen user", %{conn: conn, user: user} do
+      conn = delete(conn, Routes.user_path(conn, :delete, user))
+      assert response(conn, 204)
+
+      conn = get(conn, Routes.user_path(conn, :show, user))
+      assert response(conn, 404)
+    end
```
Comment (Rados13) [inline test/membrane_live_web/controllers/user_controller_test.exs:84]: Hmm, I am not sure if this should be possible without any authorization.

**Review by Rados13 — CHANGES_REQUESTED** [review]

File: test/support/fixtures/token_fixtures.ex [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  This module defines test helpers for creating
+  jwt mocks for testing logging of the user
+  """
+
+  def google_token_fixture(_attrs \\ %{}) do
+  end
+
+  def google_public_key_url_response() do
+  end
+end
```
[the remark this answers is not in the document]
Comment (pkrucz00) [reply test/support/fixtures/token_fixtures.ex:12]: 🗑

File: test/membrane_live/accounts_test.exs [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+      assert Accounts.list_users() == [user]
+    end
+
+    test "get_user!/1 returns the user with given uuid" do
+      user = user_fixture()
+      assert Accounts.get_user!(user.uuid) == user
+    end
+
+    test "create_user/1 with valid data creates a user" do
+      valid_attrs = %{email: "some email", name: "some name", picture: "some picture"}
```
[the remark this answers is not in the document]
Comment (pkrucz00) [reply test/membrane_live/accounts_test.exs:24]: Can I use an external library (like [ecto_fields](https://github.com/jerel/ecto_fields)) for this task? 📚

File: test/membrane_live_web/controllers/user_controller_test.exs (hunk 1 above)
Comment (pkrucz00) [reply test/membrane_live_web/controllers/user_controller_test.exs:84]: I'm not sure I understand. A logged in user has access to this endpoint. Should I remove it entirely?

Comment (pkrucz00) [reply test/membrane_live_web/controllers/user_controller_test.exs:84]: I'll comment out that only the user himself can delete his account ❌.

File: test/membrane_live/accounts_test.exs (hunk 3 above)
[the remark this answers is not in the document]
Comment (pkrucz00) [reply test/membrane_live/accounts_test.exs]: I've used ecto_fields

File: test/membrane_live/accounts_test.exs [hunk 4]
```diff
@@ -21,12 +21,16 @@ defmodule MembraneLive.AccountsTest do
...
+        email: "john@gmail.com",
+        name: "John Kowalski",
+        picture: "kowalski.img"
+      }
 
       assert {:ok, %User{} = user} = Accounts.create_user(valid_attrs)
-      assert user.email == "some email"
-      assert user.name == "some name"
-      assert user.picture == "some picture"
+      assert user.email == "john@gmail.com"
```
Comment (Rados13) [inline test/membrane_live/accounts_test.exs:31]: I think we need an additional test for checking whether an email constraint works correctly.

**Review by Rados13 — APPROVED** [review]
One test is needed.

File: test/membrane_live/accounts_test.exs (hunk 4 above)
Comment (pkrucz00) [reply test/membrane_live/accounts_test.exs:31]: Will do 👌

**Pull request merged**